contents/handbook/small-teams/product-analytics/index.mdx

```diff
@@ -17,6 +17,47 @@ hideAnchor: false
 
 <Objectives />
 
+## Who are we building for?
+
+### Personas
+
+- Primary Personas:
+    - **Product manager** (ex-engineer type)
+        - Support the product teams to build the best products. They guide the product roadmap by speaking to customers and diving into the data.
+        - Product managers are the power-users of analytics (further evidence in the data analysis of paying users). They have desire and the time to go significantly deeper into the data.
+    - **Product engineer**
+        - These are the engineers building the product. Normally full-stack engineers skewing frontend or frontend engineers. 
+        - Product engineers have more limited time. Need to quickly get high-quality insights to inform what they are building and assess what they've shipped.
+- Limited focus:
+    - Growth engineer
+- Not a focus but should be usable by:
+    - Marketing
+    - Leadership team
+
+### What types of companies?
+We should build for the highest-performing product teams (engineers, PMs, designers) building the most loved products at high-growth startups. The product analytics teams focus specific on product managers (the ex-engineer type) and product-engineers (full-stack engineers skewed towards the frontend) but it should be usable by everyone in the product team.
+
+#### Suggested criteria
+- 15-500 employees
+- $100k+/month in revenue OR very large number of consumer users
+- Raised from leading investors
+- Not yet IPO'ed
+
+### Jobs to be done
+
+Product analytics is a _wide_ tool which fulfills many job-to-be-done (non-exhaustive list):
+
+- Monitoring KPIs - how are the specific KPIs (product/team/company) doing? Are there any big changes, is everything going roughly in the right direction?
+- Insights into a new feature you've built - I've created a new feature and I want to keep track of success metrics, view live events of people, and view session recordings of how people use it (Combines feature flags, trends, funnels and session recordings)
+- Watching for errors and debugging - something went wrong (error gets trigger, visual regression, drop in conversion), getting told it went wrong, debugging it, shipping a solution and making sure that fixes it
+- Conversion optimization - the growth team is monitoring how particular KPIs are doing, trying to come up with experiments, shipping features to try and improve these
+- Answering product strategy questions - which customers should we focus on, what are our most used/valued features. e.g. should we increase the pricing from X to Y? Which customers should we focus on?
+
+You can broadly group the job-to-be-done of Product Analytics in PostHog as:
+
+- **Creating**: you know the view of the data that you are wanting to see **beforehand** and then you create the view. E.g. creating a dashboard to Monitor KPIs, or creating the funnel for your onboarding flow
```

Comment:
"you know the view of the data you want to see" sounds very abstract to me. 

Proposals:
- "You have a question in mind and use PostHog's analytics features to answer it."
- "You have a specific query in mind, you open PostHog to create it"